pkg/virt-controller/services/template.go

```diff
@@ -356,7 +356,11 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 		userId = util.NonRootUID
 	}
 
-	gracePeriodSeconds := gracePeriodInSeconds(vmi)
+	// Pad the virt-launcher grace period.
+	// Ideally we want virt-handler to handle tearing down
+	// the vmi without virt-launcher's termination forcing
+	// the vmi down.
+	gracePeriodSeconds := gracePeriodInSeconds(vmi) + int64(15)
```

Comment:
`gracePeriodSeconds` used to equal `gracePeriodInSeconds(vmi) + 15` while `gracePeriodKillAfter` used to equal `gracePeriodSeconds + int64(15) == gracePeriodInSeconds(vmi) + 15 + int64(15) == gracePeriodInSeconds(vmi) + 30`. This changes the calculated value, which leads to the failing unit tests.